launcher/ui/pages/global/LauncherPage.cpp

```diff
@@ -106,6 +106,8 @@ LauncherPage::LauncherPage(QWidget *parent) : QWidget(parent), ui(new Ui::Launch
     }
     connect(ui->fontSizeBox, SIGNAL(valueChanged(int)), SLOT(refreshFontPreview()));
     connect(ui->consoleFont, SIGNAL(currentFontChanged(QFont)), SLOT(refreshFontPreview()));
+
+    connect(ui->themeCustomizationWidget, &ThemeCustomizationWidget::currentCatChanged, this, &LauncherPage::emitCatUpdate);
```

Comment:
You should be able to connect `APPLICATION->currentCatChanged` directly

Maybe this:

```suggestion
    connect(ui->themeCustomizationWidget, &ThemeCustomizationWidget::currentCatChanged, APPLICATION, &Application::currentCatChanged);
```

Reply:
that seems to work, thank you!